src/components/molecules/Header/Sidebar.tsx

```diff
@@ -0,0 +1,72 @@
+import clsx from "clsx";
+import Link from "next/link";
+import { useState } from "react";
+import { X } from "react-feather";
+import { useTranslation } from "react-i18next";
+
+import { Divider } from "@/components/atoms";
+import Anchors from "@/components/atoms/Anchor/Anchor.css";
+import { Dropdown } from "@/components/atoms/Dropdown/Dropdown";
+import { useBoardGroups } from "@/lib/queries";
+import i18n from "@/utils/i18n";
+
+import * as styles from "./Sidebar.css";
+
+interface SidebarProps {
+  isOpened: boolean;
+  toggle: () => void;
```

Comment:
Sidebar는 닫는 기능만 수행하기 때문에 `close()`를 받도록 수정합시다.

```suggestion
  close: () => void;
```

Reply:
[766f915](https://github.com/sparcs-kaist/ara-web/pull/12/commits/766f9159e299dc517b7055a23c126633b1920572)